buildspec.yml

```diff
@@ -0,0 +1,63 @@
+version: 0.2
+
+#env:
+  #variables:
+     # key: "value"
```

Comment:
Just curious, why so much placeholder/example options commented out?

Reply:
I copied the buildspec file from warp, and it had these similar options commented out. I didn't want to remove them in case any of the options might become useful in the future, but I can remove them in favor of having cleaner code.

Reply:
I thought it may be something like that, it makes sense to keep them to me.